{
    "active_lock_reason": null,
    "assignee": null,
    "assignees": [],
    "author_association": "NONE",
    "body": "If I run the project from the GUI of the IDE (run or debug button), it will be stuck at \"Syncing files to device\", making it impossible to test the app because it's just a white screen. Not even the `main()` method is reached. If I run the profiler mode, it will run (I guess because it doesn't need to synchronize files), but the Flutter Performance view won't detect the app.\r\n\r\nIn any case, if I close the app from my phone, the \"runner\" will detect it and show that the connection was lost. Then I can normally open the app but of course not hot reload.\r\n\r\nIf I run the app from the terminal, it works well.\r\n\r\nIf I try to attach to an instance started from the terminal, it will be stuck again at \"Syncing files to device\".\r\n\r\nAlso, each time the process gets stuck, the IDE shows these internal error messages:\r\n<details>\r\n<summary>stacktrace.txt</summary>\r\n<pre>\r\n<code>\r\njava.lang.reflect.InvocationTargetException\r\n\tat jdk.internal.reflect.GeneratedMethodAccessor166.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat com.intellij.execution.process.ProcessHandler$2.invoke(ProcessHandler.java:246)\r\n\tat com.sun.proxy.$Proxy46.onTextAvailable(Unknown Source)\r\n\tat com.intellij.execution.process.ProcessHandler.notifyTextAvailable(ProcessHandler.java:220)\r\n\tat com.intellij.execution.process.BaseOSProcessHandler$SimpleOutputReader.onTextAvailable(BaseOSProcessHandler.java:154)\r\n\tat com.intellij.util.io.BaseOutputReader.sendText(BaseOutputReader.java:206)\r\n\tat com.intellij.util.io.BaseOutputReader.processInput(BaseOutputReader.java:190)\r\n\tat com.intellij.util.io.BaseOutputReader.readAvailableBlocking(BaseOutputReader.java:137)\r\n\tat com.intellij.util.io.BaseDataReader.readAvailable(BaseDataReader.java:68)\r\n\tat com.intellij.util.io.BaseDataReader.doRun(BaseDataReader.java:157)\r\n\tat com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)\r\n\tat com.intellij.util.io.BaseDataReader.lambda$start$0(BaseDataReader.java:51)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)\r\n\tat java.base/java.lang.Thread.run(Thread.java:834)\r\nCaused by: java.lang.NoClassDefFoundError: de/roderick/weberknecht/WebSocketException\r\n\tat java.base/java.lang.Class.forName0(Native Method)\r\n\tat java.base/java.lang.Class.forName(Class.java:315)\r\n\tat com.sun.proxy.$Proxy235.<clinit>(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\r\n\tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1022)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1008)\r\n\tat com.intellij.util.EventDispatcher.createMulticaster(EventDispatcher.java:77)\r\n\tat com.intellij.util.EventDispatcher.getMulticaster(EventDispatcher.java:112)\r\n\tat io.flutter.run.daemon.FlutterApp.changeState(FlutterApp.java:557)\r\n\tat io.flutter.run.daemon.FlutterAppDaemonEventListener.onAppStarted(FlutterApp.java:866)\r\n\tat io.flutter.run.daemon.DaemonEvent$AppStarted.accept(DaemonEvent.java:239)\r\n\tat io.flutter.run.daemon.DaemonEvent.dispatch(DaemonEvent.java:52)\r\n\tat io.flutter.run.daemon.DaemonApi.dispatch(DaemonApi.java:177)\r\n\tat io.flutter.run.daemon.DaemonApi$1.onTextAvailable(DaemonApi.java:148)\r\n\t... 23 more\r\nCaused by: java.lang.ClassNotFoundException: de.roderick.weberknecht.WebSocketException PluginClassLoader[PluginDescriptor(name=Flutter, id=io.flutter, path=C:\\Users\\leozx\\AppData\\Local\\JetBrains\\Toolbox\\apps\\IDEA-U\\ch-0\\203.7148.57.plugins\\flutter-intellij, version=52.2.5)] com.intellij.ide.plugins.cl.PluginClassLoader@6ff310c0\r\n\tat com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:129)\r\n\tat java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)\r\n\t... 40 more\r\n</code>\r\n</pre>\r\n</details>\r\n\r\n<details>\r\n<summary>induced.txt</summary>\r\n<pre>\r\n<code>\r\nFollowing exceptions happened soon after this one, most probably they are induced.\r\n\r\n\r\n2021-02-02T03:20:10.961\r\njava.lang.NoClassDefFoundError: de/roderick/weberknecht/WebSocketException\r\n\tat io.flutter.vmService.DartVmServiceDebugProcess.lambda$scheduleConnect$1(DartVmServiceDebugProcess.java:238)\r\n\tat com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)\r\n\tat com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)\r\n\tat com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:270)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)\r\n\tat java.base/java.lang.Thread.run(Thread.java:834)\r\nCaused by: java.lang.ClassNotFoundException: de.roderick.weberknecht.WebSocketException PluginClassLoader[PluginDescriptor(name=Flutter, id=io.flutter, path=C:\\Users\\leozx\\AppData\\Local\\JetBrains\\Toolbox\\apps\\IDEA-U\\ch-0\\203.7148.57.plugins\\flutter-intellij, version=52.2.5)] com.intellij.ide.plugins.cl.PluginClassLoader@6ff310c0\r\n\tat com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:129)\r\n\tat java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)\r\n\t... 12 more\r\n</code>\r\n</pre>\r\n</details>\r\n\r\nFinally, if I press the stop button, the phone will stay in the white screen and the IDE will show these internal errors messages:\r\n<details>\r\n<summary>stacktrace.txt</summary>\r\n<pre>\r\n<code>\r\njava.lang.reflect.InvocationTargetException\r\n\tat jdk.internal.reflect.GeneratedMethodAccessor178.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat com.intellij.execution.process.ProcessHandler$2.invoke(ProcessHandler.java:246)\r\n\tat com.sun.proxy.$Proxy46.processWillTerminate(Unknown Source)\r\n\tat com.intellij.execution.process.ProcessHandler.fireProcessWillTerminate(ProcessHandler.java:228)\r\n\tat com.intellij.execution.process.ProcessHandler.lambda$destroyProcess$0(ProcessHandler.java:122)\r\n\tat com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:279)\r\n\tat com.intellij.execution.process.ProcessHandler.destroyProcess(ProcessHandler.java:120)\r\n\tat com.intellij.execution.impl.ExecutionManagerImpl$Companion.stopProcess(ExecutionManagerImpl.kt:110)\r\n\tat com.intellij.execution.impl.ExecutionManagerImpl$Companion.stopProcess(ExecutionManagerImpl.kt:87)\r\n\tat com.intellij.execution.impl.ExecutionManagerImpl.stopProcess(ExecutionManagerImpl.kt)\r\n\tat com.intellij.execution.actions.StopAction.actionPerformed(StopAction.java:197)\r\n\tat com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:281)\r\n\tat com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:209)\r\n\tat com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:177)\r\n\tat com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:160)\r\n\tat com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:477)\r\n\tat java.desktop/java.awt.Component.processEvent(Component.java:6417)\r\n\tat java.desktop/java.awt.Container.processEvent(Container.java:2263)\r\n\tat java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)\r\n\tat java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)\r\n\tat java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)\r\n\tat java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)\r\n\tat java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)\r\n\tat java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)\r\n\tat java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)\r\n\tat java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)\r\n\tat java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)\r\n\tat java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)\r\n\tat java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)\r\n\tat java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)\r\n\tat java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)\r\n\tat java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)\r\n\tat java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)\r\n\tat java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)\r\n\tat com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:976)\r\n\tat com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:911)\r\n\tat com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:840)\r\n\tat com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)\r\n\tat com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)\r\n\tat com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)\r\n\tat com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)\r\n\tat com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:507)\r\n\tat java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)\r\n\tat java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)\r\n\tat java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)\r\n\tat java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)\r\n\tat java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)\r\n\tat java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)\r\nCaused by: java.lang.NoClassDefFoundError: Could not initialize class com.sun.proxy.$Proxy235\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\r\n\tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1022)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1008)\r\n\tat com.intellij.util.EventDispatcher.createMulticaster(EventDispatcher.java:77)\r\n\tat com.intellij.util.EventDispatcher.getMulticaster(EventDispatcher.java:112)\r\n\tat io.flutter.run.daemon.FlutterApp.changeState(FlutterApp.java:557)\r\n\tat io.flutter.run.daemon.FlutterApp.shutdownAsync(FlutterApp.java:568)\r\n\tat io.flutter.run.daemon.FlutterAppDaemonEventListener.processWillTerminate(FlutterApp.java:795)\r\n\tat io.flutter.run.daemon.DaemonApi$1.processWillTerminate(DaemonApi.java:156)\r\n\t... 54 more\r\n</code>\r\n</pre>\r\n</details>\r\n\r\n<details>\r\n<summary>induced.txt</summary>\r\n<pre>\r\n<code>\r\nFollowing exceptions happened soon after this one, most probably they are induced.\r\n\r\n\r\n2021-02-02T03:22:37.072\r\njava.lang.reflect.InvocationTargetException\r\n\tat jdk.internal.reflect.GeneratedMethodAccessor179.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat com.intellij.execution.process.ProcessHandler$2.invoke(ProcessHandler.java:246)\r\n\tat com.sun.proxy.$Proxy46.processTerminated(Unknown Source)\r\n\tat com.intellij.execution.process.ProcessHandler.lambda$notifyTerminated$2(ProcessHandler.java:204)\r\n\tat com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:279)\r\n\tat com.intellij.execution.process.ProcessHandler.notifyTerminated(ProcessHandler.java:187)\r\n\tat com.intellij.execution.process.ProcessHandler.notifyProcessTerminated(ProcessHandler.java:183)\r\n\tat com.intellij.execution.process.BaseProcessHandler.onOSProcessTerminated(BaseProcessHandler.java:61)\r\n\tat com.intellij.execution.process.OSProcessHandler.onOSProcessTerminated(OSProcessHandler.java:206)\r\n\tat com.intellij.execution.process.BaseOSProcessHandler$1.lambda$startNotified$0(BaseOSProcessHandler.java:97)\r\n\tat com.intellij.execution.process.ProcessWaitFor.lambda$new$0(ProcessWaitFor.java:55)\r\n\tat com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)\r\n\tat com.intellij.execution.process.ProcessWaitFor.lambda$new$1(ProcessWaitFor.java:33)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)\r\n\tat java.base/java.lang.Thread.run(Thread.java:834)\r\nCaused by: java.lang.NoClassDefFoundError: Could not initialize class com.sun.proxy.$Proxy235\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\r\n\tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1022)\r\n\tat java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1008)\r\n\tat com.intellij.util.EventDispatcher.createMulticaster(EventDispatcher.java:77)\r\n\tat com.intellij.util.EventDispatcher.getMulticaster(EventDispatcher.java:112)\r\n\tat io.flutter.run.daemon.FlutterApp.changeState(FlutterApp.java:557)\r\n\tat io.flutter.run.daemon.FlutterAppDaemonEventListener.processTerminated(FlutterApp.java:808)\r\n\tat io.flutter.run.daemon.DaemonApi$1.processTerminated(DaemonApi.java:161)\r\n\t... 24 more\r\n</code>\r\n</pre>\r\n</details>\r\n\r\n\r\n## Steps to Reproduce\r\n\r\nTo be honest, I have no idea how this happened. It was working well. I just decided to reboot my PC and then the error started. I was in the latest version (53.0). I decided to try the latest dev release and also the second latest stable version (52.2.5) but it still happens. I tried invalidating cache and restarting, changing flutter channels and doing upgrades, doing a lot of  `flutter clean`, restart my PC again, even downgrading IntelliJ. But the error is still present.\r\n\r\nI tried in a new \"black\" project and it still happens, so since the plugin has no configuration I think the following is valid:\r\n\r\n1. Create a new project\r\n2. Run the project from the \"Run\" button.\r\n\r\n\r\n## Version info\r\n```\r\nC:\\tools\\flutter\\bin\\flutter.bat doctor --verbose\r\n[√] Flutter (Channel beta, 1.25.0-8.3.pre, on Microsoft Windows [Version 10.0.19042.746], locale es-PE)\r\n    • Flutter version 1.25.0-8.3.pre at C:\\tools\\flutter\r\n    • Framework revision 5d36f2e7f5 (3 weeks ago), 2021-01-14 15:57:49 -0800\r\n    • Engine revision 7a8f8ca02c\r\n    • Dart version 2.12.0 (build 2.12.0-133.7.beta)\r\n\r\n[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)\r\n    • Android SDK at C:\\Users\\leozx\\AppData\\Local\\Android\\sdk\r\n    • Platform android-30, build-tools 30.0.3\r\n    • Java binary at: C:\\Users\\leozx\\.jdks\\openjdk-15.0.1\\bin\\java\r\n    • Java version OpenJDK Runtime Environment (build 15.0.1+9-18)\r\n    • All Android licenses accepted.\r\n\r\n[!] Android Studio (not installed)\r\n    • Android Studio not found; download from https://developer.android.com/studio/index.html\r\n      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).\r\n\r\n[√] IntelliJ IDEA Ultimate Edition (version 2020.3)\r\n    • IntelliJ at C:\\Users\\leozx\\AppData\\Local\\JetBrains\\Toolbox\\apps\\IDEA-U\\ch-0\\203.7148.57\r\n    • Flutter plugin version 52.2.5\r\n    • Dart plugin version 203.6912\r\n\r\n[√] Connected device (1 available)\r\n    • Pixel 3a (mobile) • 98FAY14MT6 • android-arm64 • Android 11 (API 30)\r\n\r\n! Doctor found issues in 1 category.\r\nProcess finished with exit code 0\r\n```",
    "closed_at": "2021-02-02T21:29:22Z",
    "comment_data": [
        {
            "author_association": "MEMBER",
            "body": "`Caused by: java.lang.NoClassDefFoundError: de/roderick/weberknecht/WebSocketException`\r\nThe WebSocket library is distributed with the Dart plugin. My guess is that you have a class path configuration problem. The easiest way to fix that probably is to completely uninstall all IntelliJ and Android Studio, then reinstall the latest of whichever you want. Uninstall instructions are here:\r\nhttps://www.jetbrains.com/help/idea/uninstall.html\r\n\r\nI don't know much about Windows but there is probably some system setting that controls where it looks to find applications to run. Make sure that does not include anything you uninstalled, and that it does include the one you are using. Let us know if that helps.",
            "created_at": "2021-02-02T16:29:16Z",
            "html_url": "https://github.com/flutter/flutter-intellij/issues/5236#issuecomment-771763511",
            "id": 771763511,
            "issue_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MTc2MzUxMQ==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771763511/reactions"
            },
            "updated_at": "2021-02-02T16:29:16Z",
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771763511",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/8518285?v=4",
                "events_url": "https://api.github.com/users/stevemessick/events{/privacy}",
                "followers_url": "https://api.github.com/users/stevemessick/followers",
                "following_url": "https://api.github.com/users/stevemessick/following{/other_user}",
                "gists_url": "https://api.github.com/users/stevemessick/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/stevemessick",
                "id": 8518285,
                "login": "stevemessick",
                "node_id": "MDQ6VXNlcjg1MTgyODU=",
                "organizations_url": "https://api.github.com/users/stevemessick/orgs",
                "received_events_url": "https://api.github.com/users/stevemessick/received_events",
                "repos_url": "https://api.github.com/users/stevemessick/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/stevemessick/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/stevemessick/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/stevemessick"
            }
        },
        {
            "author_association": "NONE",
            "body": "Given what you said about the Dart plugin, I tried downgrading it to 203.6803 (current is 203.6912), invalidating cache, and restarting and... it worked! I'm now with:\r\n- IntelliJ 2020.3.2\r\n- Dart Plugin 203.6912\r\n- Flutter Plugin 52.2.5\r\n\r\nI'll upgrade the plugins and see if the problem persists.",
            "created_at": "2021-02-02T17:25:31Z",
            "html_url": "https://github.com/flutter/flutter-intellij/issues/5236#issuecomment-771815123",
            "id": 771815123,
            "issue_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MTgxNTEyMw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771815123/reactions"
            },
            "updated_at": "2021-02-02T17:25:31Z",
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771815123",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/42286051?v=4",
                "events_url": "https://api.github.com/users/lsalazarm99/events{/privacy}",
                "followers_url": "https://api.github.com/users/lsalazarm99/followers",
                "following_url": "https://api.github.com/users/lsalazarm99/following{/other_user}",
                "gists_url": "https://api.github.com/users/lsalazarm99/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/lsalazarm99",
                "id": 42286051,
                "login": "lsalazarm99",
                "node_id": "MDQ6VXNlcjQyMjg2MDUx",
                "organizations_url": "https://api.github.com/users/lsalazarm99/orgs",
                "received_events_url": "https://api.github.com/users/lsalazarm99/received_events",
                "repos_url": "https://api.github.com/users/lsalazarm99/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/lsalazarm99/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/lsalazarm99/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/lsalazarm99"
            }
        },
        {
            "author_association": "NONE",
            "body": "I upgraded those plugins to their latest stable versions (the ones I had when the problem started) and **it's working now**.\r\nMany thanks!",
            "created_at": "2021-02-02T17:29:05Z",
            "html_url": "https://github.com/flutter/flutter-intellij/issues/5236#issuecomment-771818400",
            "id": 771818400,
            "issue_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MTgxODQwMA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771818400/reactions"
            },
            "updated_at": "2021-02-02T17:29:05Z",
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/771818400",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/42286051?v=4",
                "events_url": "https://api.github.com/users/lsalazarm99/events{/privacy}",
                "followers_url": "https://api.github.com/users/lsalazarm99/followers",
                "following_url": "https://api.github.com/users/lsalazarm99/following{/other_user}",
                "gists_url": "https://api.github.com/users/lsalazarm99/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/lsalazarm99",
                "id": 42286051,
                "login": "lsalazarm99",
                "node_id": "MDQ6VXNlcjQyMjg2MDUx",
                "organizations_url": "https://api.github.com/users/lsalazarm99/orgs",
                "received_events_url": "https://api.github.com/users/lsalazarm99/received_events",
                "repos_url": "https://api.github.com/users/lsalazarm99/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/lsalazarm99/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/lsalazarm99/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/lsalazarm99"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "It must have been a combination of finding a new version of the Dart plugin and rebuilding the cache that did it. I'm glad your problem is fixed. I'll close this issue.",
            "created_at": "2021-02-02T21:29:22Z",
            "html_url": "https://github.com/flutter/flutter-intellij/issues/5236#issuecomment-772002094",
            "id": 772002094,
            "issue_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MjAwMjA5NA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/772002094/reactions"
            },
            "updated_at": "2021-02-02T21:29:22Z",
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/comments/772002094",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/8518285?v=4",
                "events_url": "https://api.github.com/users/stevemessick/events{/privacy}",
                "followers_url": "https://api.github.com/users/stevemessick/followers",
                "following_url": "https://api.github.com/users/stevemessick/following{/other_user}",
                "gists_url": "https://api.github.com/users/stevemessick/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/stevemessick",
                "id": 8518285,
                "login": "stevemessick",
                "node_id": "MDQ6VXNlcjg1MTgyODU=",
                "organizations_url": "https://api.github.com/users/stevemessick/orgs",
                "received_events_url": "https://api.github.com/users/stevemessick/received_events",
                "repos_url": "https://api.github.com/users/stevemessick/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/stevemessick/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/stevemessick/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/stevemessick"
            }
        }
    ],
    "comments": 4,
    "comments_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236/comments",
    "created_at": "2021-02-02T08:54:14Z",
    "event_data": [
        {
            "actor": {
                "avatar_url": "https://avatars.githubusercontent.com/u/8518285?v=4",
                "events_url": "https://api.github.com/users/stevemessick/events{/privacy}",
                "followers_url": "https://api.github.com/users/stevemessick/followers",
                "following_url": "https://api.github.com/users/stevemessick/following{/other_user}",
                "gists_url": "https://api.github.com/users/stevemessick/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/stevemessick",
                "id": 8518285,
                "login": "stevemessick",
                "node_id": "MDQ6VXNlcjg1MTgyODU=",
                "organizations_url": "https://api.github.com/users/stevemessick/orgs",
                "received_events_url": "https://api.github.com/users/stevemessick/received_events",
                "repos_url": "https://api.github.com/users/stevemessick/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/stevemessick/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/stevemessick/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/stevemessick"
            },
            "commit_id": null,
            "commit_url": null,
            "created_at": "2021-02-02T16:29:28Z",
            "event": "labeled",
            "id": 4280287626,
            "label": {
                "color": "000000",
                "name": "waiting for customer response"
            },
            "node_id": "MDEyOkxhYmVsZWRFdmVudDQyODAyODc2MjY=",
            "performed_via_github_app": null,
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/events/4280287626"
        },
        {
            "actor": {
                "avatar_url": "https://avatars.githubusercontent.com/u/26350515?v=4",
                "events_url": "https://api.github.com/users/no-response%5Bbot%5D/events{/privacy}",
                "followers_url": "https://api.github.com/users/no-response%5Bbot%5D/followers",
                "following_url": "https://api.github.com/users/no-response%5Bbot%5D/following{/other_user}",
                "gists_url": "https://api.github.com/users/no-response%5Bbot%5D/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/apps/no-response",
                "id": 32965360,
                "login": "no-response[bot]",
                "node_id": "MDM6Qm90MzI5NjUzNjA=",
                "organizations_url": "https://api.github.com/users/no-response%5Bbot%5D/orgs",
                "received_events_url": "https://api.github.com/users/no-response%5Bbot%5D/received_events",
                "repos_url": "https://api.github.com/users/no-response%5Bbot%5D/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/no-response%5Bbot%5D/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/no-response%5Bbot%5D/subscriptions",
                "type": "Bot",
                "url": "https://api.github.com/users/no-response%5Bbot%5D"
            },
            "commit_id": null,
            "commit_url": null,
            "created_at": "2021-02-02T17:25:34Z",
            "event": "unlabeled",
            "id": 4280619543,
            "label": {
                "color": "000000",
                "name": "waiting for customer response"
            },
            "node_id": "MDE0OlVubGFiZWxlZEV2ZW50NDI4MDYxOTU0Mw==",
            "performed_via_github_app": null,
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/events/4280619543"
        },
        {
            "actor": {
                "avatar_url": "https://avatars.githubusercontent.com/u/8518285?v=4",
                "events_url": "https://api.github.com/users/stevemessick/events{/privacy}",
                "followers_url": "https://api.github.com/users/stevemessick/followers",
                "following_url": "https://api.github.com/users/stevemessick/following{/other_user}",
                "gists_url": "https://api.github.com/users/stevemessick/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/stevemessick",
                "id": 8518285,
                "login": "stevemessick",
                "node_id": "MDQ6VXNlcjg1MTgyODU=",
                "organizations_url": "https://api.github.com/users/stevemessick/orgs",
                "received_events_url": "https://api.github.com/users/stevemessick/received_events",
                "repos_url": "https://api.github.com/users/stevemessick/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/stevemessick/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/stevemessick/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/stevemessick"
            },
            "commit_id": null,
            "commit_url": null,
            "created_at": "2021-02-02T21:29:22Z",
            "event": "closed",
            "id": 4281800401,
            "node_id": "MDExOkNsb3NlZEV2ZW50NDI4MTgwMDQwMQ==",
            "performed_via_github_app": null,
            "state_reason": null,
            "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/events/4281800401"
        }
    ],
    "events_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236/events",
    "html_url": "https://github.com/flutter/flutter-intellij/issues/5236",
    "id": 799072258,
    "labels": [],
    "labels_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "MDU6SXNzdWU3OTkwNzIyNTg=",
    "number": 5236,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236/reactions"
    },
    "repository_url": "https://api.github.com/repos/flutter/flutter-intellij",
    "state": "closed",
    "state_reason": "completed",
    "timeline_url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236/timeline",
    "title": "Stuck at \"Syncing files to device\" when running the project from \"Run\" or \"Debug\" button",
    "updated_at": "2021-02-02T21:29:23Z",
    "url": "https://api.github.com/repos/flutter/flutter-intellij/issues/5236",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/42286051?v=4",
        "events_url": "https://api.github.com/users/lsalazarm99/events{/privacy}",
        "followers_url": "https://api.github.com/users/lsalazarm99/followers",
        "following_url": "https://api.github.com/users/lsalazarm99/following{/other_user}",
        "gists_url": "https://api.github.com/users/lsalazarm99/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/lsalazarm99",
        "id": 42286051,
        "login": "lsalazarm99",
        "node_id": "MDQ6VXNlcjQyMjg2MDUx",
        "organizations_url": "https://api.github.com/users/lsalazarm99/orgs",
        "received_events_url": "https://api.github.com/users/lsalazarm99/received_events",
        "repos_url": "https://api.github.com/users/lsalazarm99/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/lsalazarm99/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/lsalazarm99/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/lsalazarm99"
    }
}